src/integrationTest/kotlin/com/sourcegraph/cody/edit/DocumentCodeTest.kt

```diff
@@ -12,13 +12,15 @@ import com.sourcegraph.cody.edit.widget.LensSpinner
 import com.sourcegraph.cody.edit.widget.LensWidgetGroup
 import com.sourcegraph.cody.util.CodyIntegrationTextFixture
 import com.sourcegraph.cody.util.CustomJunitClassRunner
+import kotlin.test.Ignore
 import org.hamcrest.Matchers.startsWith
 import org.junit.Assert.assertThat
 import org.junit.Test
 import org.junit.runner.RunWith
 
 @RunWith(CustomJunitClassRunner::class)
 class DocumentCodeTest : CodyIntegrationTextFixture() {
+  @Ignore
```

Comment:
What is the reason of that? Is it too flaky?

Reply:
yup